### PR TITLE
Add ESLint and JSCS

### LINKED ISSUE #17: Add jscs
Use jscs-preset-gulp

### LINKED ISSUE #18: Explicit eslint usage
To bring inline with gulp repos. Use eslint-config-gulp and update code style

### LINKED ISSUE #19: Update repo location references
In package.json, readme, etc

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #17 
Fixes #18 
FIxes #19

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 language: node_js
+
+sudo: false
+
 node_js:
+  - 'stable'
+  - '0.12'
   - '0.10'
+
 script:
```
Comment (phated) [inline .travis.yml]: might as well remove the whole script since lint isn't part of it anymore

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 language: node_js
+
+sudo: false
```
Comment (phated) [inline .travis.yml]: I prefer sudo: false at the top of the file (just a nitpick)

**Comment by pdehaan** [comment]
Done.

I don't see Travis complaining/applauding on this page. Not sure if you still need to enable it for this org/repo.

**Comment by phated** [comment]
I've tried toggling travis, can you push something here and see if it is fixed?

**Comment by pdehaan** [comment]
Done. :+1:

**Pull request merged**